corelib/include/rtabmap/core/odometry/OdometryCuVSLAM.h

```diff
@@ -0,0 +1,42 @@
+#ifndef ODOMETRYCUVSLAM_H_
+#define ODOMETRYCUVSLAM_H_
+
+#include <rtabmap/core/Odometry.h>
+#include <cuvslam/cuvslam.h>
```

Comment:
Ideally, we would not want to have this header in the public interface of rtabmap. `#include <cuvslam/cuvslam.h>` would only be in the cpp.

If classes or functions of CuVSLAM used below can be forward declared, that would be better to do so.

See for example what we did for OdometryOpenVINS: https://github.com/introlab/rtabmap/blob/master/corelib/include/rtabmap/core/odometry/OdometryOpenVINS.h